prow/plank/controller.go

```diff
@@ -412,14 +412,15 @@ func guberURL(pj kube.ProwJob, build string) string {
 	var url string
 	if pj.Spec.Type == kube.PresubmitJob || pj.Spec.Type == kube.BatchJob {
 		url = guberBasePR
+		if pj.Spec.Refs.Org != "kubernetes" {
```

Comment:
This is wrong. we still need this logic for push jobs. 

Can you add controller_test.go to exercise the different cases of this function?